src/main/java/com/cgdpd/library/validation/Validator.java

```diff
@@ -21,6 +21,19 @@ public static String requiredNotBlank(String paramName, String value) {
         return value;
     }
 
+    public static String requiredValidIsbn13(String paramName, String value) {
```

Comment:
I meant the method `requiredPositive` and `requiredNotNegative` put this in the bottom